core/src/main/java/nu/yona/server/properties/SecurityProperties.java

```diff
@@ -22,7 +22,7 @@
 	private String ovpnProfileFile;
 
 	/**
-	 * If true, Cross Origin Resource Sharing is allowed. This is necessary for Swagger UI.
+	 * If true, Cross Origin EntityModel Sharing is allowed. This is necessary for Swagger UI.
```

Comment:
COEMS? Never heard of...

Reply:
:) This is a funny one caused by the Spring HATEOAS migration script.
I'll correct it.